openlibrary/templates/type/tag/view.html

```diff
@@ -0,0 +1,8 @@
+$def with (tag)
+
+$var title: $tag.name
+
+
+<div>
+    Tags View Page
```

Comment:
The ⛔ sign ___below___ is because the GitHub editor expects all text files to end with one blank line.